SoftLayer/CLI/user/edit_notifications.py

```diff
@@ -0,0 +1,34 @@
+"""Enable or Disable specific noticication for the current user"""
+# :license: MIT, see LICENSE for more details.
+
+import click
+
+import SoftLayer
+from SoftLayer.CLI import environment
+
+
+@click.command()
+@click.option('--enable/--disable', default=True,
+              help="Enable (DEFAULT) or Disable selected notification")
+@click.argument('notification', nargs=-1, required=True)
+@environment.pass_env
+def cli(env, enable, notification):
+    """Enable or Disable specific notifications.
```

Comment:
Mention that notification names need quotes around them, something like this.
`Enable or Disable specific notifications for the active user. Notification names should be enclosed in quotation marks.`

Reply:
Done